src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { User } from '../../types/user';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
```

Comment:
Don't forget about destructure (user) if you use more than 1 property from object